ledger/acctupdates.go

```diff
@@ -1658,7 +1682,7 @@ func (au *accountUpdates) getCreatorForRoundImpl(rnd basics.Round, cidx basics.C
 
 		// If this is the most recent round, au.creatables has the latest
 		// state and we can skip scanning backwards over creatableDeltas
-		if offset == uint64(len(au.deltas)) {
+		if offset == uint64(len(au.creatableDeltas)) {
```

Comment:
while this won't break anything, it's inconsistent to compare the offset ( which is calculated by the len(au.deltas) ) with the len(au.creatableDeltas). It's true that len(au.deltas) == len(au.creatableDeltas), but if you'd like to use the au.creatableDeltas, then please use it all over the function. i.e. lines 1677 and 1724 as well.